src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -39,6 +46,20 @@ public void afterCompletion(
 
     // skip logging the status endpoint
     if (!url.endsWith("/status")) {
+
+      Map<String, Object> stackDriverPayload = new HashMap<>();
+      if ("post".equalsIgnoreCase(method)) {
```

Comment:
We also have `PUT` requests.  Also, these should probably use the shared constants (e.g. the `org.springframework.web.bind.annotation.RequestMethod` enum)

Reply:
...or we can just always log